### PR TITLE
Fixes #106: Remove redundant alphabetical listing of monsters from toctree

### LINKED ISSUE #106: sidebar nav duplicates a-z list of monsters

### PR DESCRIPTION
**Pull request by eepMoody**

### CODE REVIEW

**Comment by maxrothman** [comment]
Adding reference to issue from PR for future us: #106

File: source/monsters/index.rst [hunk 1]
```diff
@@ -20,26 +20,16 @@ Monsters
...
-    :glob:
-    :maxdepth: 1
-
-    Monster Descriptions <monsters_a-z/index>
-
-------------------------------------------
-
-Monster Descriptions
-=====================
+.. container:: h1
```
Comment (maxrothman) [inline source/monsters/index.rst]: Why this and not an actual header?

File: source/monsters/index.rst [hunk 2]
```diff
@@ -20,26 +20,16 @@ Monsters
...
-
-      monsters_a-z/*/index
-      Miscellaneous Creatures <miscellaneous-creatures/index>
-      Nonplayer Characters <nonplayer-characters/index>
+        :name: monster-descs
+        :glob:
+        :titlesonly:
+        :maxdepth: 3
+
+        Monster Descriptions <monsters_a-z/index>
```
Comment (maxrothman) [inline source/monsters/index.rst]: Why the explicit reference to the index rather than just a glob?

File: source/monsters/monsters_a-z/A/index.rst [hunk 3]
```diff
@@ -5,5 +5,5 @@ A
    :titlesonly:
    :glob:
 
-   */index
+   **/index
```
Comment (maxrothman) [inline source/monsters/monsters_a-z/A/index.rst]: Why this change?

Side note: we should really standardize how we do this. It's different in a bunch of places. We could probably just stick `**` in here and be good.

File: source/monsters/monsters_a-z/index.rst [hunk 4]
```diff
@@ -1,12 +1,16 @@
...
+.. container:: columns
+
+    .. toctree::
+      :name: monsterlist
+      :titlesonly:
+      :glob:
+
+      */index
+      Miscellaneous Creatures <../miscellaneous-creatures/index>
+      Nonplayer Characters <../nonplayer-characters/index>
```
Comment (maxrothman) [inline source/monsters/monsters_a-z/index.rst]: If we want these to appear here, do we want to just move them?

File: _themes/sass/_theme_type.sass [hunk 5]
```diff
@@ -0,0 +1,236 @@
+// Typography
```
Comment (maxrothman) [inline _themes/sass/_theme_type.sass]: What does adding this file do?

Comment (eepMoody) [inline _themes/sass/_theme_type.sass]: Allows me to custom style the typography instead of having it be part of a bower dependency.

File: source/monsters/index.rst (hunk 1 above)
Comment (eepMoody) [inline source/monsters/index.rst]: If it was, it would appear in the TocTree and the bug wouldn't be fixed.

File: source/monsters/index.rst (hunk 2 above)
Comment (eepMoody) [inline source/monsters/index.rst]: If I did a glob of all indexes, they would appear in the toctree and the bug wouldn't be fixed.

File: source/monsters/monsters_a-z/A/index.rst (hunk 3 above)
Comment (eepMoody) [inline source/monsters/monsters_a-z/A/index.rst]: It's different depending on whether you want all subdirectories or just one, I believe? In this case I wanted all of them.

File: source/monsters/monsters_a-z/index.rst (hunk 4 above)
Comment (eepMoody) [inline source/monsters/monsters_a-z/index.rst]: We could, but it'd break outside links. I don't see any reason to just leave them as-is.

File: source/monsters/index.rst (hunk 2 above)
Comment (eepMoody) [inline source/monsters/index.rst]: To be more specific, I _think_ that this essentially references the structure included in that index file. If I globbed them, it would build a new instance of that structure, which would make it appear twice in the toctree.

File: source/monsters/monsters_a-z/A/index.rst (hunk 3 above)
Comment (maxrothman) [inline source/monsters/monsters_a-z/A/index.rst]: Right, but there's only 1 more level under A (it's Angels)

Comment (eepMoody) [inline source/monsters/monsters_a-z/A/index.rst]: Ah, i see. It must have been from when i was trying to get it to render sub-indexes. Still never got it working.

File: source/monsters/monsters_a-z/index.rst (hunk 4 above)
Comment (maxrothman) [inline source/monsters/monsters_a-z/index.rst]: I agree that we shouldn't break outside links unnecessarily, but I also don't want that concern to become something that drags us down. If it's a strong concern we should figure out an automated way of addressing it, e.g. a better 404 page, auto-created redirects, etc. Part of the whole reason we chose Sphinx was that it allowed us to move things without breaking the site.

**Comment by maxrothman** [comment]
Given the magnitude and visibility of this bug, :+1: regardless of the comments. We can address them later.
